third_party/internal/godocfx/main.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*Command godocfx generates DocFX YAML for Go code.
+
+Usage:
+
+    godocfx [flags] path
+
+    cd module && godocfx ./...
+    godocfx cloud.google.com/go/...
+    godocfx -print cloud.google.com/go/storage/...
+    godocfx -out custom/output/dir cloud.google.com/go/...
+    godocfx -rm cloud.google.com/go/...
+
+See:
+* https://dotnet.github.io/docfx/spec/metadata_format_spec.html
+* https://github.com/googleapis/doc-templates
+* https://github.com/googleapis/doc-pipeline
+
+TODO:
+  * Cross link referenced packages.
+*/
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"gopkg.in/yaml.v2"
+)
+
+func main() {
+	print := flag.Bool("print", false, "Print instead of save (default false)")
+	rm := flag.Bool("rm", false, "Delete out directory before generating")
+	outDir := flag.String("out", "obj/api", "Output directory (default obj/api)")
+	flag.Parse()
+	if flag.NArg() != 1 {
+		log.Fatalf("%s expected 1 arg, got %d", os.Args[0], flag.NArg())
```

Comment:
Nit: could this error message be improved? `%s missing required argument: module path`